<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>500</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>File editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>6</y>
           <width>339</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Poppins Medium</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Find</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="find"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Poppins Medium</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Replace </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="replace"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>173</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="listWidgetDrop" name="listWidget">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDropMode::DropOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::DropAction::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="statuscontainer">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="exportfolder">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
         </font>
        </property>
        <property name="text">
         <string>Export folder</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="convert">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
         </font>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MailAttachment"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanup">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
         </font>
        </property>
        <property name="text">
         <string>Clean selection</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditDelete"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addfiles">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
         </font>
        </property>
        <property name="text">
         <string>Add Files</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentPageSetup"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>297</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>listWidgetDrop</class>
   <extends>QListWidget</extends>
   <header location="global">listwidgetdrop.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
